schemas/oic.r.coapcloudconf-schema.json

```diff
@@ -1,43 +1,38 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
-  "id": "https://www.openconnectivity.org/ocf-apis/core/schemas/oic.r.coapcloudconf-schema.json#",
+  "id": "https://www.openconnectivity.org/ocf-apis/core-extensions/schemas/oic.r.coapcloudconf-schema.json#",
   "definitions": {
     "oic.r.coapcloudconf": {
       "type": "object",
       "properties": {
         "apn": {
           "type": "string",
-          "description": "Authentication provider's name issuing the auth code",
+          "description": "The Authorisation Provider through which an Access Token was obtained.",
           "pattern": "^.*$"
         },
         "cis": {
           "type": "string",
-          "description": "Indicates URL for CoAP native cloud interface server to be registered",
+          "description": "URL of OCF Cloud",
           "format": "uri"
         },
         "at": {
           "type": "string",
-          "description": "Indicates access token which is given in return of auth code issued by an appropriate account server",
-          "pattern": "^.*$"
-        },
-        "sid": {
-          "type": "string",
-          "description": "The Server identity of the OCF Cloud used for verifying the OCF Cloud certificate.",
+          "description": "Access Token which is returned by an Authorisation Provider or OCF Cloud.",
           "pattern": "^.*$"
         },
         "clec": {
-          "enum": [0, 1, 2, 3, 4, 5, 6, 7, 255],
-          "description": "Indicates a failure reason (0: No Error. 1: Failed to access CoAP Cloud server,  2: No response from CoAP Cloud server,  3: Invalid Access Token, 4: Failed to refresh Access Token, 5: Failed to find a registered device in CoAP Cloud, 6: Failed to find a registered user in CoAP Cloud, 7~254: Reserved, 255: Unknown error)",
+          "enum": [0, 1, 2, 3, 255],
+          "description": "Last Error Code during Cloud Provisioning (0: No Error, 1: Failed to access OCF Cloud server, 2: No response from OCF Cloud server, 3: Failed to refresh Access Token, 4~254: Reserved, 255: Unknown error)",
```

Comment:
Be sure that these descriptions match the latest CCC CR.  I think 2 maps to two different errors now (i.e. no response and 4.xx response).  See the CR for confirmation.

Reply:
4.xx response is from Cloud to Device. This value will be set in the clec for Mediator to retrieve from Device.

Reply:
Yes, I get that.  What I'm saying is that the descriptions in this "description" do not 100% match the descriptions from the CR.  They should match.  Look at the CR and have the descriptions here match them.

Reply:
I synchronized this text with the text in table (Table 3 oic.r.coapcloudconf Resource Type definition). I am not sure if we should put in detailed conditions in here. It might become too verbose.